### DISCUSSION TITLE
How to specify an intermediate certificate in meta-rauc?

### DISCUSSION THREAD

**Discussion by DanielNofal** [rauc/meta-rauc#325] [Q&A] [answered]
In meta-rauc; It is possible to specify a keyring, certificate and private key in site.conf, but how to specify also an intermediat certificate?

**Comment by ejoerns** [comment, marked as answer]
@HishamNofal  This should be possible via `BUNDLE_ARGS`.

For prior discussion about an explicit option, see https://github.com/rauc/meta-rauc/issues/183
